tools/mmp/driver.cs

```diff
@@ -442,6 +442,29 @@ static void Main2 (string [] args)
 
 			}
 
+			if (IsUnified) {
+				List<ExclusionItem> exclusionItems = ReadExclusionList ();
+				List<string> referencesToFix = new List<string> ();
+
+				foreach (var asm in references) {
+					List<ExclusionItem> possibleMatches = exclusionItems.Where (x => x.Name == Path.GetFileName (asm)).ToList ();
+					if (possibleMatches.Count > 0) {
+						var module = ModuleDefinition.ReadModule (asm);
+						if (possibleMatches.Any (x => x.GUID == module.Mvid))
+							referencesToFix.Add (asm);
+					}
+				}
+				if (referencesToFix.Count > 0) {
+					string facadeRoot = string.Format ("/Library/Frameworks/Xamarin.Mac.framework/Versions/Current/lib/mono/{0}/Facades/", IsUnifiedMobile ? "Xamarin.Mac" : "4.5");
```

Comment:
Should this use `.../Current/...` or can it the get the path in a better manner?